eclass/autotools.eclass

```diff
@@ -61,21 +68,21 @@ inherit libtool
 # version.
 # If a newer slot is stable on any arch, and is NOT reflected in this list,
 # then circular dependencies may arise during emerge @system bootstraps.
-# 
-# See bug 312315 and 465732 for further information and context.
-# 
+#
+# See bug #312315 and bug #465732 for further information and context.
+#
 # Do NOT change this variable in your ebuilds!
 # If you want to force a newer minor version, you can specify the correct
 # WANT value by using a colon:  <PV>:<WANT_AUTOMAKE>
 _LATEST_AUTOMAKE=( 1.16.2-r1:1.16 )
 
 _automake_atom="sys-devel/automake"
 _autoconf_atom="sys-devel/autoconf"
-if [[ -n ${WANT_AUTOMAKE} ]]; then
+if [[ -n ${WANT_AUTOMAKE} ]] ; then
 	case ${WANT_AUTOMAKE} in
 		# Even if the package doesn't use automake, we still need to depend
 		# on it because we run aclocal to process m4 macros.  This matches
-		# the autoreconf tool, so this requirement is correct.  #401605
+		# the autoreconf tool, so this requirement is correct, bug #401605.
 		none) ;;
 		latest)
 			# Use SLOT deps if we can.  For EAPI=0, we get pretty close.
```

Comment:
I put it here because GitHub does not let me put below... Is this still required as we don't have EAPI 0 anymore?

# Use SLOT deps if we can. For EAPI=0, we get pretty close. if [[ ${EAPI:-0} != 0 ]] ; then _automake_atom="|| ( printf '>=sys-devel/automake-%s:%s ' ${_LATEST_AUTOMAKE[@]/:/ })" else _automake_atom="|| (printf '>=sys-devel/automake-%s ' ${_LATEST_AUTOMAKE[@]/%:*} )" fi ;; *) _automake_atom="=sys-devel/automake-${WANT_AUTOMAKE}*" ;;

Reply:
Good spot, thanks!